README.md

```diff
@@ -123,6 +123,17 @@ The value of Faraday `params_encoder` can be any object that responds to:
 The encoder will affect both how query strings are processed and how POST bodies
 get serialized. The default encoder is Faraday::NestedParamsEncoder.
 
+## Authentication
+  
+Basic and Token authentication are handled by Faraday::Request::BasicAuthentication and Faraday::Request::TokenAuthentication respectively. These can be added as middleware manually or through the helper methods.
+
+```ruby
+Faraday.new(...) do |conn|
+  conn.basic_auth('username', 'password')
+  conn.token_auth('authentication-token') 
+end
```

Comment:
Thanks for authoring this explanation!

Is this code example really 2 code examples? If yes, could we make it two different code samples?